kcidb/templates/fluster_test.j2

```diff
@@ -0,0 +1,26 @@
+{# Test macros #}
+{% from "common.j2" import indent, stats_indent, selected_origins_for_fluster %}
+
+{% macro container_summary(container, max_list_len) %}
+    {{- "\nFLUSTER TESTS" }}
+    {% if container.tests %}
+        {% set fluster_tests = container.tests_root["fluster"].waived_status_tests[false] %}
+        {% set waived_fluster_tests = container.tests_root["fluster"].waived_status_tests[false] %}
+        {% set updated_waived_fluster_tests = container.tests_root.slice_compatible_field(waived_fluster_tests) %}
+        {% set updated_dict = {} %}
+        {% for status, tests in updated_waived_fluster_tests.items() %}
+            {% set _ = updated_dict.update({status: tests| selectattr('origin', 'in', selected_origins_for_fluster) | groupby("environment_compatible")}) %}
+        {% endfor %}
+        {% set sorted_new = container.tests_root.sort_fluster_tests_by_platform(updated_dict) %}
+        {% set new_fluster_stats = container.tests_root.get_fluster_stats(sorted_new) %}
+        {% for platform, stats in new_fluster_stats.items() %}
+        {{- "\n" + indent + platform }}:
+            {% for test, passed_count, failed_count, total_count in stats %}
+                {{- indent + "-" + test}}: {{ " " * (stats_indent|length - indent|length - test|length) }} passed: {{ passed_count }}, failed: {{ failed_count }}, total: {{ total_count }}
```

Comment:
I think perhaps just another level of indentation would work better, as dashes are part of the test name, which makes it look confusing, like the names were truncated. Or at the very least put a space between the dash and the test name.

Reply:
I also see, that the test names all include `v4l2-decoder-conformance-`. So perhaps you can write `FLUSTER V4L2 DECODER CONFORMANCE TESTS` instead of `FLUSTER TESTS`, and remove that string from every test. It will improve readability quite a bit. Also, you could use the emojis we are using everywhere else to indicate passed and failed counts, and e.g. `Σ` for the total. And finally, you could get rid of the colons to minimize visual noise. So, this:

```
FLUSTER TESTS

      google,kingoftown:
      -v4l2-decoder-conformance-av1:                  passed: 0, failed: 0, total: 0
      -v4l2-decoder-conformance-av1-chromium-10bit:   passed: 0, failed: 0, total: 0
      -v4l2-decoder-conformance-h264:                 passed: 0, failed: 0, total: 0
      -v4l2-decoder-conformance-h264-frext:           passed: 0, failed: 0, total: 0
      -v4l2-decoder-conformance-h265:                 passed: 1, failed: 1, total: 2
      -v4l2-decoder-conformance-vp8:                  passed: 0, failed: 0, total: 0
      -v4l2-decoder-conformance-vp9:                  passed: 0, failed: 0, total: 0

```

Would become this:

```
FLUSTER V4L2 DECODER CONFORMANCE TESTS

      google,kingoftown
          av1                  ✅ 0  ❌ 0  Σ 0
          av1-chromium-10bit   ✅ 0  ❌ 0  Σ 0
          h264                 ✅ 0  ❌ 0  Σ 0
          h264-frext           ✅ 0  ❌ 0  Σ 0
          h265                 ✅ 1  ❌ 1  Σ 2
          vp8                  ✅ 0  ❌ 0  Σ 0
          vp9                  ✅ 0  ❌ 0  Σ 0
```

Reply:
This format looks much better. Thanks. I'll use it.